python/paddle/linalg.py

```diff
@@ -12,32 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .tensor import inverse as inv
+from .tensor import inverse as inv  # noqa: F401
 from .tensor.linalg import (
-    cholesky,
-    cholesky_solve,
-    cond,
-    corrcoef,
-    cov,
-    det,
-    eig,
-    eigh,
-    eigvals,
-    eigvalsh,
+    cholesky,  # noqa: F401
+    cholesky_solve,  # noqa: F401
+    cond,  # noqa: F401
+    corrcoef,  # noqa: F401
+    cov,  # noqa: F401
+    det,  # noqa: F401
+    eig,  # noqa: F401
+    eigh,  # noqa: F401
+    eigvals,  # noqa: F401
+    eigvalsh,  # noqa: F401
+    householder_product,  # noqa: F401
     lstsq,
```

Comment:
why `lstsq` not followed by `# noqa: F401`

Reply:
I haven't done anything here except add the `householder_product`, I think it is modified by pre-commit?

Reply:
I think I'd better restore this part of code